docs/srs.md

```diff
@@ -0,0 +1,130 @@
+4. <https://cutt.ly/zf7MkUX> : This article is an explanation of how to connect PostgreSQL with Flask.
+5. <https://hydra.cc/docs/intro/> : Hydra is an open-source Python framework that simplifies the development of research and other complex applications.
+
+## 2\. Overall Description
+
+### 2.1 Product perspective
+
+An accounting database system stores the following information.
+
+- **Equipment details:** It includes information about the equipment, such as laptops, monitors, mice, keyboards, headphones, which are issued to employees, as well as information about the equipment that is needed in other cities.
+- **Employee description:** It includes the employee's id, first name, last name, position, mail, phone number, city, project.
+- **Description of interaction:** It includes information which technique is given to each employee or information that the employee works on his technique. It also includes information about the workplace.
+- **Description of equipment status:**
+ it includes information about the equipment that needs to be repaired or replaced by a new one.
+- **Software descriptions:**
+It includes information about the software, such as product key, quantity, expiration date, cost, date of purchase and who owns .
+### 2.2 Product features
+
+This item presents the DFD-diagram, which shows the general cooperation in the system. They are presented in pictures 1.1-1.2 ![image1.png](images/image1.png)
+
+Picture 1.1 - DFD diagram A-0 level.
+
+![image2.png](images/image2.png)
+
+Picture 1.2 - DFD diagram A0 level.
+
+### 2.3 Operating environment
+
+This product will work in any browser, as it will represent a web page. On PostgreSQL 9.5, Python 3.8\. This database will be located on Amazon AWS servers.
+
+### 2.4 Design and implementation constraints
+
+Our constraints are:
+
+- Possibilities of Python programming language and PostgreSQL DBMS;
+- Writing Python code that satisfies PEP 8;
+- Constraints that may be imposed by the project business logic such as:
+
+  - The ability to have multiple owners for one piece of equipment/furniture
+  - The database is operated in two modes: full access and partial (read-only) access, etc.
+
+  ## 3\. System features
+
+  ### 3.1 Description and priority
+
+- **Import/Export** to .csv file/ .xls file: saving or uploading tables.
+
+- **Generating QR codes** with unique inventory number: unique number on the inventory, which will make it easier to count and keep records
+
+- **Back up**: Saving the database before entering and changing information.
+
+- **Data Structuring**: all information in a digital repository is organized into a clear hierarchical structure with records, tables or objects.
+- **Multi-user Access**: provides multiple users access to all types of information stored in one and the same data store.
+- **Add/Remove** information about employee's or equipment in the database.
+- **User-Friendly**: read-only access to the database.
```

Comment:
Do you consider 'user-friendly' as a feature or as a quality?  Is it actually should appear in the Feature description section?

Reply:
Ok, we will move it to other sections